README.md

```diff
@@ -71,17 +71,18 @@ description: |
   This is a collection of NOMAD schemas for simulation data.
 ```
 
-and define the ```nomad.yaml``` configuration file of your NOMAD instance in the root folder with the following content:
+and register the plugin in the ```nomad.yaml``` configuration file of your NOMAD root folder:
 
 ```yaml
 plugins:
-  include:
```

Comment:
Flease, we can leave this here. It is just about probably adding the comment that you can find in the nomad.yaml if you feel so.

Furthermore, it seems that manually installing nomad-lab[parsing] fixes the built-in plugins.

Reply:
Well, this is the only correction. I haven't seen any other errors, and have verified that it works...
You can ask Lauri to take a look.

Reply:
Just noticed this comment now:

> Please, we can leave this here. It is just about probably adding the comment that you can find in the nomad.yaml if you feel so

No, this setup disables ALL other plugins, which should never be the default! There are only a few scenarios where this is wishful.
I do think that this has not been properly explained in the general docs. I'll see to amend that.

Reply:
Just to double-check, @JFRudzinski or @Bernadette-Mohr, following the old setup, can you still run the regular parsers?
Not the tests, but the actual parsers.